src/White.jsx

```diff
@@ -0,0 +1,174 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCircle } from "@fortawesome/free-solid-svg-icons";
+
+export default function White(props) {
+  const { AllshoesName, AllImage } = props;
+
+  return (
+    <>
+      <div className="container">
+        <div className="listContainer">
+          {AllshoesName.map((ele) => {
+            if (ele.color === "WHITE") {
+              return (
+                <div className="infoCard">
+                  <img alt="shoes" width="100px" src={AllImage[ele.id]} />
+                  <p className="name">{ele.name}</p>
+                  <p className="color">
+                    {ele.color}{" "}
+                    <FontAwesomeIcon
+                      icon={faCircle}
+                      size="lg"
+                      color={ele.color}
+                    />
+                  </p>
+                  <p className="type">{ele.type}</p>
+                  <p className="year">{ele.where_to_bought}</p>
+                  <p className="where">{ele.year_bounght}</p>
+                </div>
+              );
+            } else {
+              return null;
+            }
+          })}
+        </div>
+      </div>
+      <div className="container">
+        <div className="listContainer">
+          {AllshoesName.map((ele) => {
+            if (ele.color === "RED") {
+              return (
+                <div className="infoCard">
+                  <img alt="shoes" width="100px" src={AllImage[ele.id]} />
+                  <p className="name">{ele.name}</p>
+                  <p className="color">{ele.color}</p>
+                  <p className="type">{ele.type}</p>
+                  <p className="year">{ele.where_to_bought}</p>
+                  <p className="where">{ele.year_bounght}</p>
+                  <p>
+                    <FontAwesomeIcon
+                      icon={faCircle}
+                      size="lg"
+                      color={ele.color}
+                    />
+                  </p>
+                </div>
```

Comment:
All of these are almost exactly the same! You copied the code for every color - I would make a component that makes all of this, and then see if you can change your code so you don't need to have the same code for every single color. Remember DRY - Don't Repeat Yourself.

For example, if you decided to change the design, or change the text, you need to make the same change in 7 places! That's a waste of time and also it's really easy to forget to make a change for one of them.